src/py42/clients/securitydata.py

```diff
@@ -20,15 +19,13 @@ def __init__(
         self._preservation_data_service = preservation_data_service
         self._saved_search_service = saved_search_service
         self._storage_service_factory = storage_service_factory
-        self._client_cache = {}
-        self._client_cache_lock = Lock()
 
     @property
     def savedsearches(self):
         """A collection of methods related to retrieving forensic search data.
 
         Returns:
-            :class: `py42._internal.services.SavedSearchService`
+            :class: `py42._internal.services.savedsearch.SavedSearchService`
```

Comment:
The `_internal` part is still there and not needed (old path)

Reply:
Oh whoops, totally missed that part the first time